c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -400,6 +400,27 @@ typedef enum {
 /** Battery is having hardware error or end of life. */
 #define LSM_BATTERY_STATUS_ERROR                    1 << 7
 
+#define LSM_VOLUME_WRITE_CACHE_POLICY_UNKNOWN       1
+#define LSM_VOLUME_WRITE_CACHE_POLICY_WRITE_BACK    2
+#define LSM_VOLUME_WRITE_CACHE_POLICY_AUTO          3
+#define LSM_VOLUME_WRITE_CACHE_POLICY_WRITE_THROUGH 4
+
+#define LSM_VOLUME_WRITE_CACHE_STATUS_UNKNOWN       1
+#define LSM_VOLUME_WRITE_CACHE_STATUS_WRITE_BACK    2
+#define LSM_VOLUME_WRITE_CACHE_STATUS_WRITE_THROUGH 3
+
+#define LSM_VOLUME_READ_CACHE_POLICY_UNKNOWN        1
+#define LSM_VOLUME_READ_CACHE_POLICY_ENABLED        2
+#define LSM_VOLUME_READ_CACHE_POLICY_DISABLED       3
+
+#define LSM_VOLUME_READ_CACHE_STATUS_UNKNOWN        1
+#define LSM_VOLUME_READ_CACHE_STATUS_ENABLED        2
+#define LSM_VOLUME_READ_CACHE_STATUS_DISABLED       3
+
+#define LSM_VOLUME_PHYSICAL_DISK_CACHE_UNKNOWN      1
+#define LSM_VOLUME_PHYSICAL_DISK_CACHE_ENABLED      2
+#define LSM_VOLUME_PHYSICAL_DISK_CACHE_DISABLED     3
+#define LSM_VOLUME_PHYSICAL_DISK_CACHE_USE_DISK_SETTING     4
 
```

Comment:
I'm wondering if we could reduce the number of constants, so that we don't need to have a separate one for each?  Basically have:

```C
#define LSM_VOLUME_CACHE_UNKNOWN  1
#define LSM_VOLUME_CACHE_ENABLED  2
#define LSM_VOLUME_CACHE_DISABLED  3
#define LSM_VOLUME_CACHE_AUTO  4
#define LSM_VOLUME_CACHE_WRITE_BACK  5
#define LSM_VOLUME_CACHE_WRITE_THROUGH  6
#define LSM_VOLUME_CACHE_USE_DISK_SETTING  7
```

What do people think about that?